src/InvocationModel.ts

```diff
@@ -104,6 +104,10 @@ export class InvocationModel implements coreTypes.InvocationModel {
             }
         }
 
+        if (!response.returnValue && response.outputData.length == 0) {
```

Comment:
I tested this on an http trigger with no output binding and it _is_ returning the value. It literally just shows the json as the response which seems pretty weird
<img width="432" alt="Screen Shot 2022-10-07 at 2 14 08 PM" src="https://user-images.githubusercontent.com/11282622/194653426-cf7f2c84-56b8-44db-bedb-7d111b6e91bb.png">

In other words, I do think we have to be more careful on this. Possible solutions include
1. Only set return value here based on trigger (either "not http" or "is durable"). This would match old behavior
2. Ask durable to pass in a flag somewhere saying "returnIfNoBindingSet". This would require changes in both repos
3. Ask durable to configure an output binding under the covers for the user. This would only require changes in durable

Reply:
I would favor the first option since it matches old behavior anyway. Option 3 isn't really feasible because there isn't a valid output binding type that Durable can set

Reply:
I'm okay with the first option to unblock durable, although I would still prefer if we had a better reason than "it matches old behavior". The old behavior had a lot of weird things that did not make sense

Reply:
Went with the first option to unblock Durable and added a detailed comment on why this is useful